<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LogicPower2" name="tests.uaua.TestServicesRasschetnue" tests="10" failures="0" timestamp="20 Aug 2015 09:22:45 GMT" time="803.029" errors="1">
  <testcase name="testBBR01" time="182.155" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testSTP01" time="21.240" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testBM01" time="379.342" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testRDR01" time="38.931" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testRPMP01" time="25.041" classname="tests.uaua.TestServicesRasschetnue">
    <error type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at tests.uaua.TestServicesRasschetnue.testRPMP01(TestServicesRasschetnue.java:211)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:345)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1009)
]]>
    </error>
  </testcase> <!-- testRPMP01 -->
  <testcase name="testKP01" time="22.936" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testAKB01" time="38.273" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testPDG01" time="25.438" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testSOP01" time="48.679" classname="tests.uaua.TestServicesRasschetnue"/>
  <testcase name="testAKB02" time="20.994" classname="tests.uaua.TestServicesRasschetnue"/>
</testsuite> <!-- tests.uaua.TestServicesRasschetnue -->
